### PR TITLE
Adding debug logs to elan and upping flair logs to errors

### PR DESCRIPTION
**Pull request by Hamishpk**

### CODE REVIEW

File: elan/rpc/gc.go [hunk 1]
```diff
@@ -83,9 +83,9 @@ func (s *server) deleteAll(ctx context.Context, blobs []*ppb.Blob, hard bool) er
 	var me *multierror.Error
 	for _, blob := range blobs {
 		key := blob.CachePrefix + blob.Hash
-		log.Notice("Deleting blob %s", key)
 		if exists, err := s.bucket.Exists(ctx, key); exists {
 			if err := s.bucket.Delete(ctx, key, hard); err != nil {
+				log.Error("Error deleting blob: %v/home/hpitkeathly/.local/state/nvim/swap//%home%hpitkeathly%please-servers%elan%rpc%rpc.go.swp", err)
```
Comment (Birkalo) [inline elan/rpc/gc.go:88]: this looks like a copy&paste typo, but also might be something important, i have no idea

File: flair/trie/replication.go [hunk 2]
```diff
@@ -84,9 +84,9 @@ func (r *Replicator) SequentialAck(key string, f ReplicatedAckFunc) error {
 		}
 		merr = multierror.Append(merr, err)
 		if i < r.Replicas-1 {
-			log.Debug("Error reading from replica for %s: %s. Will retry on next replica.", key, err)
+			log.Error("Error reading from replica for %s: %s. Will retry on next replica.", key, err)
```
Comment (Birkalo) [inline flair/trie/replication.go:87]: nit: line 87, 89, 131 could have the `err` verb as  `%w`

File: elan/rpc/gc.go (hunk 1 above)
Comment (Hamishpk) [reply elan/rpc/gc.go]: :aha: You're right that is a c&p error. That's the location of a previous vim swp file I needed to remove before I could open a new one.

File: flair/trie/replication.go (hunk 2 above)
Comment (Hamishpk) [reply flair/trie/replication.go]: Good spot. I have changed these to %v. Githubs UI isn't great here but you can see the changes if you hit Files changed.

**Review by Birkalo — APPROVED** [review]

**Pull request merged**